content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -151,7 +151,7 @@ using the following feature gates:
 You will still have to enable each option using the `CPUManagerPolicyOptions` kubelet option.
 
 The following policy options exist for the static `CPUManager` policy:
-* `full-pcpus-only` (beta, visible by default) (1.22 or higher)
+* `full-pcpus-only` (GA, visible by default) (1.22 or higher)
```

Comment:
```suggestion
* `full-pcpus-only` (GA, visible by default) (1.33 or higher)
```

Reply:
we should mention 1.33 instead of v1.23, as `full-pcpus-only` will be in beta.

Reply:
thanks! I think I addressed in the last upload